SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Reporting Security Issues
```

Comment:
<label>**praise** </label>[non-blocking]: <subject> Great to add this document in & follow best open security practices 🎉  </subject>